content/_partials/card.html.haml

```diff
@@ -0,0 +1,25 @@
+- authorId = page.author || ''
+- author = site.authors[authorId] || OpenStruct.new({:name => authorId})
+
+-if author
+  %div{:class => "author-card", :onclick => "location.href='#{author_link(authorId)}'"}
+    %a.author.name{:href => author_link(authorId)}
+      = author.name
+    = partial("avatar.html.haml", :author => author, :authorId => authorId)
+    %ul.author.social-media-buttons
```

Comment:
Would be nice to move it to partial to DRY. https://github.com/jenkins-infra/jenkins.io/blob/master/content/_partials/author.html.haml

Reply:
@oleg-nenashev thanks for the feedback. I moved the social links to a partial, I hope that's a good trade-off between DRY and flexibility of the layouts.